src/colorizer/constants.ts

```diff
@@ -7,6 +7,8 @@ export const CANVAS_BACKGROUND_COLOR_DEFAULT = 0xf7f7f7;
 export const OUTLINE_COLOR_DEFAULT = 0xff00ff;
 export const OUTLIER_COLOR_DEFAULT = 0xc0c0c0;
 export const OUT_OF_RANGE_COLOR_DEFAULT = 0xdddddd;
+export const EDGE_COLOR_DEFAULT = 0x000000;
+export const EDGE_COLOR_ALPHA_DEFAULT = 0.25;
```

Comment:
Default color is like 25% black. I think the increased contrast looks good against the default white background, but  it usually needs to be adjusted against black backgrounds.